### PR TITLE
ptbr 374

### PR DESCRIPTION
**Pull request by BomberBeetle**

### CODE REVIEW

**Comment by BomberBeetle** [comment]
Oh, hey. I ended up also pushing a commit to my repo with new transcript overlays for pages 5 and 6 (also renaming the one for page 4 so it has the .json extension). also corrected a minor goof on one of the portuguese pages haha

**Comment by ryanvilbrandt** [comment]
> Oh, hey. I ended up also pushing a commit to my repo with new transcript overlays for pages 5 and 6 (also renaming the one for page 4 so it has the .json extension). also corrected a minor goof on one of the portuguese pages haha

Awesome! :D Happy to see this being used

File: 5/Português (Brasil).json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "width":96,
+        "text": "*luvas térmicas!*"
+    },
+    {
+        "top":105,
+        "left":545,
+        "height":35,
+        "width":96,
+        "text": "*luvas térmicas!*"
+    },
```
Comment (ryanvilbrandt) [inline 5/Português (Brasil).json:29]: I see duplicate speech balloon entries here and in Page 6 below. Do you know if this is user error, or is this possibly caused by a bug in the transcript builder?

Comment (BomberBeetle) [reply 5/Português (Brasil).json:29]: oh oops yeah that's a mistake on my part. i forgot about the builder 😅

Comment (ryanvilbrandt) [reply 5/Português (Brasil).json:29]: Cool! Good to know! 👍 If you wouldn't mind fixing that on your end first, then I'll be happy to merge this PR.

Comment (BomberBeetle) [reply 5/Português (Brasil).json]: fixed :)

File: 6/Português (Brasil).json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        "width":72,
+        "text":"Obrigado, mãe."
+    },
+    {
+        "top":1026,
+        "left":279,
+        "height":66,
+        "width":72,
+        "text":"Obrigado, mãe."
+    },
```
Comment (ryanvilbrandt) [inline 6/Português (Brasil).json:64]: One more duplicate bubble here

Comment (BomberBeetle) [reply 6/Português (Brasil).json]: fixed and also snuck in the new page

**Pull request merged**